monkestation/strings/random_crimes.txt

```diff
@@ -59,3 +59,16 @@ Reached Skate Nirvana, Flubbed Up
 Too Cool
 Cooking Reality-Bending Food
 Recycling
+Caused a Intergalactic War Between NanoTrasen And The Clown Union, Due To Adding Silentium To a Banana
+Stun Prodded a Clown And Spacing Said Clown Out a Airlock
+Blew Up a Station With a Supermatter Delamination
+Murderboned TOO HARD
+Drugged a Station's Crew With Floorpills
+Has a Engine Addiction that Causes More Harm Then Good
+Unleashed a Chaos Dunk On 15 Different Space Stations, They Were Stopped by a Security Officer On One Station Before They Did Another Chaos Dunk On Said Station, Many Know Them As "The Chaos Dunk Maniac"
```

Comment:
Length of this one might be too much for the actual consoles

Reply:
maybe.... we have to text to see